code/datums/supply_packs/black_market.dm

```diff
@@ -0,0 +1,1024 @@
+
+
+----
+
+*/
+
+
+
+/datum/supply_packs/contraband//base
+	name = "contraband crate"
+	contains = list(/obj/item/toy/bikehorn)
+	containertype = /obj/structure/closet/crate/trashcart
+	containername = "large crate"
+	group = "Black Market"
+	contraband = TRUE
+	hidden = TRUE
+	cost = 0
+	dollar_cost = 50
+
+/obj/structure/largecrate/black_market
+	//no special name so it can be hidden
+
+/obj/structure/largecrate/black_market/unpack()
+	for(var/obj/O in contents)
+		O.black_market_value = 0 //no nesting
+	..()
+
+/* --- SEIZED ITEMS --- */
+
+/*
+
+Non-USCM items, from CLF, UPP, colonies, etc. Mostly combat-related.
+
+*/
+
+/datum/supply_packs/contraband/seized
+	group = "Seized Items"
+	contains = list()
+
+/datum/supply_packs/contraband/seized/black_market_scanner
+	name = "black market scanner crate"
+	contains = list(/obj/item/device/black_market_scanner)
+	containername = "trash cart"
+	dollar_cost = 5
+	containertype = /obj/structure/closet/crate/trashcart
+
+/datum/supply_packs/contraband/seized/confiscated_equipment
+	name = "seized foreign equipment crate"
+	dollar_cost = 70
+	containertype = /obj/structure/largecrate/black_market/confiscated_equipment
+
+/obj/structure/largecrate/black_market/confiscated_equipment/Initialize()
+	. = ..()
+	var/rng = rand(1,7)
+	switch(rng)
+		if(1) //pmc
+			new /obj/item/clothing/under/marine/veteran/PMC(src)
+			new /obj/item/clothing/head/helmet/marine/veteran/PMC(src)
+			new /obj/item/clothing/suit/storage/marine/veteran/PMC(src)
+			new /obj/item/clothing/gloves/marine/veteran/PMC(src)
+			new /obj/item/clothing/mask/gas/PMC(src)
+			new /obj/item/storage/backpack/lightpack(src)
+		if(2) //dutch's
+			new /obj/item/clothing/head/helmet/marine/veteran/dutch(src)
+			new /obj/item/clothing/under/marine/veteran/dutch(src)
+			new /obj/item/clothing/suit/storage/marine/veteran/dutch(src)
+			new /obj/item/clothing/gloves/marine/veteran(src)
+		if(3) //pizza
+			new /obj/item/clothing/under/pizza(src)
+			new /obj/item/clothing/head/soft/red(src)
+		if(4) //clf
+			new /obj/item/clothing/under/colonist/clf(src)
+			new /obj/item/clothing/suit/storage/militia(src)
+			new /obj/item/clothing/head/militia(src)
+			new /obj/item/clothing/gloves/marine/veteran(src)
+			new /obj/item/storage/backpack/lightpack(src)
+		if(5) //upp
+			new /obj/item/clothing/head/helmet/marine/veteran/UPP(src)
+			new /obj/item/clothing/under/marine/veteran/UPP(src)
+			new /obj/item/clothing/suit/storage/marine/faction/UPP(src)
+			new /obj/item/clothing/shoes/marine/upp(src)
+			new /obj/item/clothing/gloves/marine/veteran(src)
+			new /obj/item/storage/backpack/lightpack(src)
+		if(6) //freelancer
+			new /obj/item/clothing/under/marine/veteran/freelancer(src)
+			new /obj/item/clothing/suit/storage/marine/faction/freelancer(src)
+			new /obj/item/clothing/gloves/marine/veteran(src)
+			new /obj/item/storage/backpack/lightpack(src)
+		if(7) //HEFA
```

Comment:
Remove HEFA please

Reply:
yea ok its not like the former lead and current vice lore manager designed hefa as a rogue USCM squad visible on half the maps ingame......